frontend/packages/rhoas-plugin/src/catalog/catalog-content.json

```diff
@@ -1,18 +1,14 @@
 {
-  "KafkaConnection": {
+  "RHOASService": {
```

Comment:
I see them as array without keys

Reply:
but what if someone updates a new item in the json? im not sure that we can rely on the order alone to give us the right value

Reply:
Not getting it. For both array or objects we depend on order. Just that object require `Object.values(serviceList)` to work and object fields have no actual use/value